modules/launcher/src/main/java/org/mule/module/launcher/AbstractArtifactDeploymentListener.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.launcher;
+
+import org.mule.api.MuleContext;
+import org.mule.config.bootstrap.ArtifactType;
+
+/**
+ * Default implementation of {@link ArtifactDeploymentListener}).
```

Comment:
An abstract class cannot be the default implementation of anything because it can be instantiated. It can be a base class for implementing the interface